build-logic/publishing/src/main/kotlin/build-logic.dokka.gradle.kts

```diff
@@ -0,0 +1,52 @@
+import ch.tutteli.gradle.plugins.dokka.DokkaPluginExtension
+import org.jetbrains.dokka.base.DokkaBase
+import org.jetbrains.dokka.base.DokkaBaseConfiguration
+import org.jetbrains.dokka.gradle.AbstractDokkaLeafTask
+import org.jetbrains.dokka.gradle.AbstractDokkaTask
+import org.jetbrains.dokka.gradle.DokkaTask
+import java.net.URL
+
+plugins {
+    id("build-logic.build-params")
+    id("ch.tutteli.gradle.plugins.dokka")
+}
+
+fun AbstractDokkaTask.configurePlugins() {
```

Comment:
same as in /build.gradle.kts is there a way to mitigate duplication?

Reply:
Well, you suggested using `ch.tutteli.gradle.plugins.dokka`, and I used it. In that case, I expect you could factor the logic into that plugin.

Of course, you can create `build-logic.dokka.gradle.kts` and factor Dokka configuration there, however, at this point it is more like your preference, and a lack of my interest in reverse-engineering `ch.tutteli.gradle.plugins.dokka`

Reply:
good, I am not familiar with build-logic and thought there might be a reason why it cannot be move to an own convention-plugin. Thanks for the explanation :+1: